ivy_tests/test_ivy/test_frontends/test_torch/test_nn/test_functional/test_convolution_functions.py

```diff
@@ -292,42 +585,55 @@ def test_torch_conv_tranpose1d(
 
 @handle_frontend_test(
     fn_tree="torch.nn.functional.conv_transpose2d",
-    dtype_vals=_x_and_filters(dim=2, transpose=True),
-)
+    x_f_d_df=_x_and_filters(
+        dtypes=helpers.get_dtypes("float", full=False),
+        data_format=st.sampled_from(["NHWC"]),
+        padding=st.sampled_from(["VALID", "SAME"]),
+        type="2d",
+        transpose=True,
+    ),)
 def test_torch_conv_tranpose2d(
     *,
-    dtype_vals,
-    on_device,
-    fn_tree,
+    x_f_d_df,
     frontend,
     test_flags,
+    fn_tree,
     backend_fw,
+    on_device,
 ):
-    dtype, vals, weight, bias, dilations, strides, padding, output_pad, fc = dtype_vals
+    (
+        input_dtype,
+        x,
+        filters,
+        dilation,
+        data_format,
+        stride,
+        padding,
+        output_shape,
+    ) = x_f_d_df
     dilations = 1  # ToDo: remove this when support for dilation > 1 is added
-    assume(
-        all(
-            x > 0
-            for x in _output_shape(
-                2, dilations, strides, padding, output_pad, vals.shape, weight.shape
-            )
-        )
-    )
+    # assume(
+    #     all(
+    #         x > 0
+    #         for x in _output_shape(
+    #             2, dilations, strides, padding, output_pad, vals.shape, weight.shape
+    #         )
+    #     )
+    # )
     helpers.test_frontend_function(
-        input_dtypes=dtype,
+        input_dtypes=input_dtype,
         backend_to_test=backend_fw,
         frontend=frontend,
         test_flags=test_flags,
         fn_tree=fn_tree,
         on_device=on_device,
-        input=vals,
-        weight=weight,
-        bias=bias,
-        stride=strides,
+        input=x,
+        weight=filters,
+        strides=stride,
         padding=padding,
-        output_padding=output_pad,
-        groups=fc,
-        dilation=dilations,
+        output_shape=output_shape,
+        data_format=data_format,
+        dilations=dilation,
```

Comment:
The arguments passed by the test should match the actual arguments of the original function as well.